### PR TITLE
fix: conversion of Ekubo x128 to Wad

### PR DESCRIPTION
**Pull request by milancermak**
Ekubo oracle price gives us already squared price (unlike the Core contract), so there's no need to do extra math. PR updates the way how we convert an x128 value to Wad - we just need to scale it up and div by 2^128.

### CODE REVIEW

File: src/utils/math.cairo [hunk 1]
```diff
@@ -49,20 +49,9 @@ pub fn div_u128_by_ray(lhs: u128, rhs: Ray) -> u128 {
 // x128 value needs to be scaled up by the quote token's decimals
 // https://docs.ekubo.org/integration-guides/reference/reading-pool-price
```
Comment (tserg) [inline src/utils/math.cairo:50]:
```suggestion
// See https://docs.ekubo.org/integration-guides/reference/reading-pool-price
// for conversion of x128 values from the Ekubo Core. However, take note that 
// Ekubo oracle extension already gives the squared price.
```
Added more explanatory notes for future reference

File: src/utils/math.cairo [hunk 2]
```diff
@@ -49,20 +49,9 @@ pub fn div_u128_by_ray(lhs: u128, rhs: Ray) -> u128 {
...
 pub fn scale_x128_to_wad(n: u256, decimals: u8) -> Wad {
```
Comment (tserg) [inline src/utils/math.cairo:51]: What do you think about renaming this to `ekubo_oracle_price_to_wad`?

**Review by tserg — COMMENTED** [review]
Looks good, I left some suggestions. There is a failing test in Receptor which needs to be updated as well.

**Comment by milancermak** [comment]
Addressed comments.

Should be mergeable if CI goes green.

File: src/utils/math.cairo (hunk 1 above)
Comment (tserg) [reply src/utils/math.cairo]: I think this was omitted. I will push this change and proceed to merge.

**Review by tserg — APPROVED** [review]

**Pull request merged**